src/pages/bookCatalog/View-bookCatalog.jsx

```diff
@@ -0,0 +1,133 @@
+import { Link } from "@radix-ui/themes";
+import React, { useState, useEffect } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+
+
+function ViewBookCatalog() {
+  const [books, setData] = useState([]);
+  const {id} = useParams();
+  const navigate =useNavigate();
+  
+
+
+  useEffect(() => {
+    const token = localStorage.getItem("token");
+
+
+     fetch(`http://localhost:3000/api/books/`+id, {
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    })
+      .then((response) => {
+        if (!response.ok) throw new Error("Network response was not ok");
+        return response.json();
+      })
+      .then((data) => {
+        console.log("Data", data);
+        setData(data);
+      })
+      .catch((error) => {
+        console.error("Error", error);
+      });
+  }, []);
+
+  const handleDelete = (id) => {
+    const userConfirmed = window.confirm("Would you like to delete?");
+    if (userConfirmed) {
+      const token = localStorage.getItem("token");
+      navigate("/book-catalog");
+
+      fetch(`http://localhost:3000/api/books/` + id, {
+        method: "DELETE",
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+      })
+        .catch((err) => console.error(err));
+    }
+  };
+  
+  return (
+    <div className="container mx-auto ">
+      <h1 className="text-2xl font-bold  pb-5">Book Catalog Information</h1>
+      <div className="flex justify- mt-4 py-4">
+      <button
+          className="bg-gray-500 text-white py-2 px-5 rounded-lg mr-4"
+        >
+          Back
```

Comment:
Should create Button.jsx component for any type of button and then just past the props